superset/views/database/views.py

```diff
@@ -156,48 +157,45 @@ def form_post(self, form: CsvToDatabaseForm) -> Response:
         ).name
 
         try:
-            utils.ensure_path_exists(config["UPLOAD_FOLDER"])
+            utils.ensure_path_exists(app.config["UPLOAD_FOLDER"])
             upload_stream_write(form.csv_file.data, uploaded_tmp_file_path)
 
             con = form.data.get("con")
             database = (
                 db.session.query(models.Database).filter_by(id=con.data.get("id")).one()
             )
 
-            # More can be found here:
-            # https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html
-            csv_to_df_kwargs = {
-                "sep": form.sep.data,
-                "header": form.header.data if form.header.data else 0,
-                "index_col": form.index_col.data,
-                "mangle_dupe_cols": form.mangle_dupe_cols.data,
-                "skipinitialspace": form.skipinitialspace.data,
-                "skiprows": form.skiprows.data,
-                "nrows": form.nrows.data,
-                "skip_blank_lines": form.skip_blank_lines.data,
-                "parse_dates": form.parse_dates.data,
-                "infer_datetime_format": form.infer_datetime_format.data,
-                "chunksize": 1000,
-            }
-            if form.null_values.data:
-                csv_to_df_kwargs["na_values"] = form.null_values.data
-                csv_to_df_kwargs["keep_default_na"] = False
-
-            # More can be found here:
-            # https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_sql.html
-            df_to_sql_kwargs = {
-                "name": csv_table.table,
-                "if_exists": form.if_exists.data,
-                "index": form.index.data,
-                "index_label": form.index_label.data,
-                "chunksize": 1000,
-            }
-            database.db_engine_spec.create_table_from_csv(
+            chunks = pd.read_csv(
                 uploaded_tmp_file_path,
-                csv_table,
+                chunksize=1000,
+                encoding="utf-8",
+                header=form.header.data if form.header.data else 0,
+                index_col=form.index_col.data,
+                infer_datetime_format=form.infer_datetime_format.data,
+                iterator=True,
+                keep_default_na=not form.null_values.data,
+                mangle_dupe_cols=form.mangle_dupe_cols.data,
+                na_values=form.null_values.data if form.null_values.data else None,
+                nrows=form.nrows.data,
+                parse_dates=form.parse_dates.data,
+                sep=form.sep.data,
+                skip_blank_lines=form.skip_blank_lines.data,
+                skipinitialspace=form.skipinitialspace.data,
+                skiprows=form.skiprows.data,
+            )
+
+            df = pd.concat(chunk for chunk in chunks)
```

Comment:
This should also work?

```python
df = pd.concat(chunks)
```

Reply:
@ktmud if we had `pylint` fully enabled I _believe_ it would catch these unnecessary statements.